### PR TITLE
More translation in FR & PT

### PR DESCRIPTION
**Pull request by baptisteArno**
Related to #210

### CODE REVIEW

**Comment by dfernandesbsolus** [comment]
I can revise the translation into Portuguese, if you want to approve the PR I will send another one later with corrections only for the translations. But as I said, my translation would be more PT-BR, which in many cases is very different from PT-PT.

**Comment by baptisteArno** [comment]
@dfernandesbsolus Ok, PT-BR is what we want I think because Typebot is use by a lot of Brazilians.

Do you confirm that the current language is ok for production, beside not being PT-BR? Or are there any weird translations?

File: apps/builder/src/locales/pt.ts [hunk 1]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'account.myAccount.emailInput.disabledTooltip':
+    'A atualização do e-mail não está disponível. Entre em contato com o suporte se quiser alterá-lo.',
+  'account.myAccount.emailInput.label': 'Endereço de e-mail:',
+  'account.myAccount.nameInput.label': 'Nome:',
+  'analytics.viewsLabel': 'Visualizações',
+  'analytics.startsLabel': 'Inícios',
+  'analytics.completionRateLabel': 'Taxa de conclusão',
+  'auth.signin.heading': 'Entrar',
+  'auth.signin.noAccountLabel.preLink': 'Não tem uma conta?',
+  'auth.signin.noAccountLabel.link': 'Registe-se gratuitamente',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:72]: The right way to write it in pt-BR is `Registre-se gratuitamente`

File: apps/builder/src/locales/pt.ts [hunk 2]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'auth.register.heading': 'Criar uma conta',
+  'auth.register.alreadyHaveAccountLabel.preLink': 'Já tem uma conta?',
+  'auth.register.alreadyHaveAccountLabel.link': 'Entrar',
+  'auth.error.default': 'Tente entrar com uma conta diferente.',
+  'auth.error.email':
+    'E-mail não encontrado. Tente entrar com um fornecedor diferente.',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:78]: If this line makes reference to different auth providers, the word `fornecedor` must be changed to `provedor`.

File: apps/builder/src/locales/pt.ts [hunk 3]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'auth.error.oauthNotLinked':
+    'Para confirmar sua identidade, entre com a mesma conta que você usou originalmente.',
+  'auth.error.unknown': 'Ocorreu um erro. Tente novamente.',
+  'auth.signinErrorToast.title': 'Não autorizado',
+  'auth.signinErrorToast.description': 'As inscrições estão desativadas.',
+  'auth.noProvider.preLink': 'Você precisa',
+  'auth.noProvider.link':
+    'configurar pelo menos um fornecedor de autenticação (E-mail, Google, GitHub, Facebook ou Azure AD).',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:86]: If this line makes reference to different auth providers, the word `fornecedor` must be changed to `provedor`.

File: apps/builder/src/locales/pt.ts [hunk 4]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'auth.emailSubmitButton.label': 'Enviar',
+  'auth.magicLink.title': 'Um email de link mágico foi enviado. 🪄',
+  'auth.magicLink.description': 'Certifique-se de verificar sua pasta de spam.',
+  'auth.socialLogin.githubButton.label': 'Continuar com GitHub',
+  'auth.socialLogin.googleButton.label': 'Continuar com Google',
+  'auth.socialLogin.facebookButton.label': 'Continuar com Facebook',
+  'auth.socialLogin.azureButton.label': 'Continuar com {azureProviderName}',
+  'auth.socialLogin.gitlabButton.label': 'Continuar com {gitlabProviderName}',
+  'auth.socialLogin.customButton.label': 'Continuar com {customProviderName}',
+  'billing.billingPortalButton.label': 'Portal de faturação',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:97]: In pt-BR, the word `faturação` is not much used. The word `cobrança` is closer to en-US word `billing`.

File: apps/builder/src/locales/pt.ts [hunk 5]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+    'Precisa de limites personalizados? Recursos específicos?',
+  'billing.customLimit.link': 'Vamos conversar!',
+  'billing.upgradeLimitLabel':
+    'Você precisa atualizar sua assinatura para {type}',
+  'billing.currentSubscription.heading': 'Assinatura',
+  'billing.currentSubscription.subheading':
+    'Assinatura atual do espaço de trabalho:',
+  'billing.currentSubscription.cancelLink': 'Cancelar minha assinatura',
+  'billing.currentSubscription.billingPortalDescription':
+    'Precisa alterar o método de pagamento ou as informações de faturação? Acesse seu portal de faturação:',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:113]: In pt-BR, the word `faturação` is not much used. The word `cobrança` is closer to en-US word `billing`.

File: apps/builder/src/locales/pt.ts [hunk 6]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'billing.pricingCard.chatsTooltip':
+    'Um chat é contado sempre que um usuário inicia uma discussão. Ele é independente do número de mensagens que ele envia e recebe.',
+  'billing.pricingCard.storageLimit': 'GB de armazenamento',
+  'billing.pricingCard.storageLimitTooltip':
+    'Você acumula armazenamento para cada arquivo que seu usuário carrega em seu bot. Se você excluir o resultado, ele liberará espaço.',
+  'billing.pricingCard.starter.description':
+    'Para indivíduos e pequenas empresas.',
+  'billing.pricingCard.starter.includedSeats': '2 assentos incluídos',
+  'billing.pricingCard.starter.brandingRemoved': 'Marca removida',
+  'billing.pricingCard.starter.fileUploadBlock': 'Bloco de entrada de arquivo',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:139]: `Bloco de entrada de arquivo` should be replaced by `Bloco de envio de arquivo`

File: apps/builder/src/locales/pt.ts [hunk 7]
```diff
@@ -21,4 +25,142 @@ export default defineLocale({
...
+  'billing.usage.chats.resetInfo': '(reiniciado todo dia 1)',
+  'billing.usage.storage.heading': 'Armazenamento',
+  'billing.usage.storage.alert.soonReach':
+    'Seus typebots são populares! Você logo alcançará o limite de armazenamento de seu plano. 🚀',
+  'billing.usage.storage.alert.updatePlan':
+    'Certifique-se de atualizar seu plano para continuar coletando arquivos enviados. Você também pode excluir arquivos para liberar espaço.',
+  'billing.limitMessage.brand': 'remover a marca',
+  'billing.limitMessage.customDomain': 'adicionar domínios personalizados',
+  'billing.limitMessage.analytics': 'desbloquear análises aprofundadas',
+  'billing.limitMessage.fileInput': 'usar blocos de entrada de arquivo',
```
Comment (erickriva) [inline apps/builder/src/locales/pt.ts:163]: `usar blocos de entrada de arquivo` should be replaced by `usar blocos de envio de arquivo`

**Comment by erickriva** [comment]
@baptisteArno I made my suggestions to fix some terms in pt-BR, as comments to `pt.ts` file.

**Pull request merged**